AUTHORS.md

```
@@ -0,0 +1,5 @@
+<!-- SPDX-FileCopyrightText: 2025 The geany contributors.
+     SPDX-License-Identifier: MPL-2.0
+-->
+
+- @AlphaOne1
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)